### DISCUSSION TITLE
Is this project abandoned?

### DISCUSSION THREAD

**Discussion by bring-shrubbery** [graphql-rust/juniper#938] [Q&A] [answered]
The last release was 4 years ago, and there are a lot of open issues on this project. Also the version never reached v1.0. Is it abandoned or is it just that stable? 🥸

**Comment by bring-shrubbery** [comment, marked as answer]
Never mind, saw the commit log. I'm new to rust, so I didn't check the cargo.io releases either :D

**Reply by t-eckert** [reply]
Don't worry, you are not alone. I almost passed this project over because the releases made it look abandoned. Looks like people are working hard on it.

**Reply by t-eckert** [reply]
Also given how the GraphQL website uses that data to show stats: 
![image](https://user-images.githubusercontent.com/29112081/120911606-89ac3080-c63d-11eb-8335-cef420a8a984.png)

**Reply by LegNeato** [reply]
I think now that we have github actions going automating posting to releases should be possible.

**Reply by bring-shrubbery** [reply]
@LegNeato I think I'll do that actually when I have free time :D
